apiserver/plane/bgtasks/issue_activites_task.py

```diff
@@ -1428,12 +1526,58 @@ def issue_activity(
                     )
 
             # Bulk create notifications
-            Notification.objects.bulk_create(bulk_notifications, batch_size=100)
+            Notification.objects.bulk_create(
+                bulk_notifications, batch_size=100)
+
+            # Add Mentioned as Issue Subscribers
+            IssueSubscriber.objects.bulk_create(mention_subscribers, batch_size=100)
+            
+            mention_notifications = []
+
+            # Send Notifications to the Mentioned and Add Mentioned as Subscribers
+            for mention_id in new_mentions:
+                for issue_activity in issue_activities_created:
+                    if (issue_activity.verb == "created" or issue_activity.verb == "updated"):
+                        mention_notifications.append(
+                            Notification(
+                                workspace=project.workspace,
+                                sender="in_app:issue_activities",
```

Comment:
Here can we add an addtional identifier `in_app:issue_activities:mention`

Reply:
I've modified it ✌🏼